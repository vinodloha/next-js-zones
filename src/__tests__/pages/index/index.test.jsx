import React from 'react'
import renderer from 'react-test-renderer'
import App from '../../../pages/index'

describe('Pages:Homepage', () => {
  it('renders the homepage', () => {
    const component = renderer.create(<App />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
