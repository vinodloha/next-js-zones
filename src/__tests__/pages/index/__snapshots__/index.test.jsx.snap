// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages:Homepage renders the homepage 1`] = `
<div
  className="jsx-3583494339 text-center"
>
  <p
    className="jsx-3583494339"
  >
    Welcome to the boilerplate React app using NextJS!
  </p>
  <p
    className="jsx-3583494339"
  >
    Try navigating to
    <a
      href="/example/second-page"
      onClick={[Function]}
      onMouseEnter={[Function]}
    >
      another page
    </a>
    and observe the URL changing
  </p>
</div>
`;
